<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тесты</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_create_test">
      <property name="text">
       <string>Создать тест</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pass_test">
      <property name="text">
       <string>Пройти тест</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_view_results">
      <property name="text">
       <string>Посмотреть результаты</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
